packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -351,15 +355,28 @@ export class IosSimulatorDevice extends DeviceBase {
 
     await this.terminateAnyRunningApplications();
 
-    await exec("xcrun", [
+    if (this.nativeLogsOutputChannel) {
+      this.nativeLogsOutputChannel.dispose();
+    }
```

Comment:
I think it is fine to just not create the channel if one already exists. We can still clear it regardless

Reply:
Changed in: https://github.com/software-mansion/radon-ide/pull/742/commits/8b9e8dfc74c040bc8c9405b0330e87b0668af9b2